File: backend/src/main/java/com/voiz/dto/SalesAnalyticsDto.java
package com.voiz.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SalesAnalyticsDto {
    
    // 선택한 시간 그룹 
    private String granurality; 
    
    // 해당 시간 그룹의 총 매출
    private Double  totalSales;

    
}

File: frontend/app/src/main/java/com/voyz/datas/model/dto/SalesAnalyticsDto.kt
package com.voyz.datas.model.dto

import com.google.gson.annotations.SerializedName

data class SalesAnalyticsDto(
    @SerializedName("granurality") val label: String?, // 백엔드에서 granurality로 보내고 있음
    @SerializedName("totalSales") val totalSales: Float
)
